sycl/include/CL/sycl/id.hpp

```diff
@@ -33,8 +33,9 @@ template <int dimensions = 1> struct id : public detail::array<dimensions> {
          range_size)
       : base(range_size.get(0)) {}
 
-  template <int N = dimensions>
-  id(typename std::enable_if<(N == 1), const item<dimensions> &>::type item)
+  template <int N = dimensions, bool with_offset = true>
+  id(typename std::enable_if<(N == 1),
```

Comment:
I think `enable_if_t` can be used in this file.

Reply:
ok